CAFAna/Core/PRISMReweightableSpectrum.cxx

```diff
@@ -0,0 +1,234 @@
+#include "CAFAna/Core/PRISMReweightableSpectrum.h"
```

Comment:
We discussed making the improvements here to Core/ReweightableSpectrum as a separate PR